### ISSUE TITLE
qgsEditError on Win10, QGIS 3.16.4, IRMT 3.7.2, OQ-Engine 3.7.1

### ISSUE THREAD

**Issue by ptormene** [gem/oq-irmt-qgis#769]
A user of a recent risk workshop experienced the following error, using the software specified in the object:

![1615223401322blob](https://user-images.githubusercontent.com/350930/111127042-ac193b00-8573-11eb-8ec3-cbdea85d0263.jpg)
![1615223256103blob](https://user-images.githubusercontent.com/350930/111127051-ad4a6800-8573-11eb-8f5c-2421cd8bcbef.jpg)

The OQ-Engine calculation producing the output that causes the problem is the following:
[M2_P2_Ejercicio.zip](https://github.com/gem/oq-irmt-qgis/files/6139948/M2_P2_Ejercicio.zip)

**Comment by ptormene** [comment]
I tried to run the calculation on a Windows 10 VM and to load the Average Asset Damages output. I did not change any setting in the following dialog and pressed Ok. Everything continued smoothly, producing the desired layer.

However, while testing this, I found another unrelated error:
```python
An error has occurred while executing Python code: 

AttributeError: 'ConnectionError' object has no attribute 'split' 
Traceback (most recent call last):
  File "C:/Users/jenkins/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\drive_oq_engine_server_dialog.py", line 186, in on_reconnect_btn_clicked
    self.attempt_login()
  File "C:/Users/jenkins/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\drive_oq_engine_server_dialog.py", line 206, in attempt_login
    self.check_engine_compatibility()
  File "C:/Users/jenkins/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\drive_oq_engine_server_dialog.py", line 217, in check_engine_compatibility
    self.engine_version = engine_version.split('-')[0]
AttributeError: 'ConnectionError' object has no attribute 'split'


Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] 
QGIS version: 3.16.4-Hannover Hannover, 654e76b3cb 

Python Path:
C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python
C:/Users/jenkins/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/jenkins/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/PROGRA~1/QGIS3~1.16/apps/qgis-ltr/./python/plugins
C:\PROGRA~1\QGIS3~1.16\apps\Python37
C:\PROGRA~1\QGIS3~1.16\apps\Python37\Scripts
C:\Program Files\QGIS 3.16\bin\python37.zip
C:\PROGRA~1\QGIS3~1.16\apps\Python37\DLLs
C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib
C:\Program Files\QGIS 3.16\bin
C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages
C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\win32
C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\win32\lib
C:\PROGRA~1\QGIS3~1.16\apps\Python37\lib\site-packages\Pythonwin
C:/Users/jenkins/AppData/Roaming/QGIS/QGIS3\profiles\default/python
```

**Comment by ptormene** [comment]
The issue was fixed and an unofficial version 3.7.3 was prepared.
